### PR TITLE
user/v-jandrew/adding-note-docking-in-responsive-viewport-mode

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**
changing date, to make the pr

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/device-mode/index.md [hunk 1]
```diff
@@ -56,10 +56,14 @@ By default the Device Toolbar opens in Responsive Viewport Mode.
...
 ### Responsive Viewport Mode
 
-To quickly test the look and feel of your page across multiple screen sizes, drag the handles to resize the viewport to your required dimensions.  You can also enter specific values in the width and height boxes.  In the following figure, the width is set to `626` and the height is set to `516`.
+To quickly test the look and feel of your page across multiple screen sizes, drag the handles to resize the viewport to your required dimensions.  You can enter any numeric values in the width and height boxes.  If you select a size larger than available in the browser window, the viewport will be auto zoomed-out to accommodate for the larger viewport.
+
+In the following figure, the width is set to `626` and the height is set to `516`.
 
 ![The handles for changing the dimensions of the viewport when in Responsive Viewport Mode.](../media/device-mode-toggle-device-toolbar-handles-highlighted.msft.png)
 
+Remember if you need more space on your screen, you can always edit the position of DevTools as explained in ![Change DevTools placement (Undock, Dock to bottom, Dock to left)](/microsoft-edge/devtools-guide-chromium/customize/placement).
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/device-mode/index.md:65]:
```suggestion
If you need more space on your screen, you can always change the position of DevTools as explained in [Change DevTools placement (Undock, Dock to bottom, Dock to left)](/microsoft-edge/devtools-guide-chromium/customize/placement).
```

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/device-mode/index.md:65]: I made 3 proposals above:

1. Removed the "Remember" part of the sentence. I'm open to discussion on this one. But it feels to me like saying "Remember" is an assumption that they have read other docs, and I don't think we can make that assumption.
2. I changed "edit" with "change". It feels more natural to me.
3. I removed the `!` character in front of the link. Links in markdown use the following syntax: `[]()`, with no `!`. If you add a `!` then it becomes an image.

File: microsoft-edge/devtools-guide-chromium/device-mode/index.md [hunk 2]
```diff
@@ -56,10 +56,14 @@ By default the Device Toolbar opens in Responsive Viewport Mode.
 
 ### Responsive Viewport Mode
 
-To quickly test the look and feel of your page across multiple screen sizes, drag the handles to resize the viewport to your required dimensions.  You can also enter specific values in the width and height boxes.  In the following figure, the width is set to `626` and the height is set to `516`.
+To quickly test the look and feel of your page across multiple screen sizes, drag the handles to resize the viewport to your required dimensions.  You can enter any numeric values in the width and height boxes.  If you select a size larger than available in the browser window, the viewport will be auto zoomed-out to accommodate for the larger viewport.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/device-mode/index.md:59]:
```suggestion
To quickly test the look and feel of your page across multiple screen sizes, drag the handles to resize the viewport to your required dimensions.  You can enter any numeric values in the width and height boxes.  If you select a size larger than available in the browser window, the viewport will be automatically scaled to accommodate for the larger viewport.
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]
Just a couple of minor comments, but otherwise looks good to me.

**Comment by JasonAndrewWriter** [comment]
> Just a couple of minor comments, but otherwise looks good to me.

I accepted the comments so looks ready. I will let Reexa know.

**Review by JasonAndrewWriter — COMMENTED** [review]
i made Patrick's changes.  :)

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**